core/modules/flowgen.cc

```diff
@@ -14,8 +14,12 @@
 
 typedef std::pair<uint64_t, struct flow *> Event;
 typedef std::priority_queue<Event, std::vector<Event>,
-                            std::function<bool(Event, Event)>>
-    EventQueue;
+                            std::function<bool(Event, Event)>> EventQueue;
+
+const Commands FlowGen::cmds = {
+  {"update", "FlowGenArg", MODULE_CMD_FUNC(&FlowGen::CommandUpdate), 1}
```

Comment:
I actually think it might be fine to stop the world -- the packet generator is just for testing and all of our measurements pause for a few seconds to wait for the generator to ramp up/down to the new load. 

Happy to implement RCU if you think it's important not to stop, but otherwise, can I just mark it as .mt_safe == 0?